src/Form/Form.js

```diff
@@ -45,9 +43,6 @@ class Form {
         this.form = form
         this.matching = matching || createMatching()
         this.formAnalyzer = new FormAnalyzer(form, input, matching)
-        this.isLogin = this.formAnalyzer.isLogin
-        this.isSignup = this.formAnalyzer.isSignup
-        this.isHybrid = this.formAnalyzer.isHybrid
```

Comment:
These now are all getters, so if we change the `formAnalyzer` results, the changes are reflected here.